### DISCUSSION TITLE
How to add a Image to a Layout screen

### DISCUSSION THREAD

**Discussion by alejandroGonGon** [orchidsoftware/platform#2033] [Q&A] [answered]
¡Hey!
 
I need help, because after searching in the doc and trying in differents ways I don't know how to add a Image to a `Layout::table` and I already done it before with the example of the doc :
 
code:
![image](https://user-images.githubusercontent.com/56725406/147491372-70ececa9-2b40-4255-af40-39090b95ff03.png)
result:
![image](https://user-images.githubusercontent.com/56725406/147491411-a7df8bd0-1802-4b91-864b-edd792800902.png)
 
And I want to add the same image but on another screen and I don't know how to do it with orchid.
 
I want something like this
![image](https://user-images.githubusercontent.com/56725406/147491611-8aeb8747-25db-4986-8c00-a77034bdb2ee.png)
This is my code : 
![image](https://user-images.githubusercontent.com/56725406/147491337-d9a60031-8a7a-4f89-af7a-616ab6fdf542.png)
 
And using that code it throws me this error.
And it throw me this error:
![image](https://user-images.githubusercontent.com/56725406/147491081-1f2fa85f-e8a4-4b75-ac8f-b55476e14eee.png)
 
And the Doc said that I can use TD in a Layout:
![image](https://user-images.githubusercontent.com/56725406/147491130-78de397f-7ac2-4554-99e5-00e7b28bc303.png)
 
Any idea to add an Image alone or something similar in a Screen ?

**Comment by tabuna** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/56725406/147491611-8aeb8747-25db-4986-8c00-a77034bdb2ee.png)

Hi, judging by the screenshot above, your layout looks like this:

```php
    public function layout(): array
    {
        return [
            Layout::legend(...),
            Layout::rows(...)
     ]
}
```

You can also insert an image into the legend using the control `Sight :: make (..)->render(...)`. Or from above using your own Blade template:

```php
    public function layout(): array
    {
        return [
            Layout::view('yourBlade'),
            Layout::legend(...),
            Layout::rows(...)
     ]
}
```

for reference you can look at this section of the documentation https://orchid.software/en/docs/custom-template/#views

**Reply by alejandroGonGon** [reply]
Again, thank you man!
